refgram/syntax/index.html

```diff
@@ -4002,158 +4002,158 @@ <h4 id="speech_act_pattern"><a href="#speech_act_pattern">Speech act pattern</a>
 <p>This is identical in meaning to:</p>
 
 <div class="gloss">
-	geo /ni (Y+AO) .<br/>
+	󱛃󱚴󱛍󱛃 󱚵󱛊󱚹 󱛘󱚿󱚺󱛎󱛃󱛙 󱛕<br/>
 	Geo ní chao.<br/>
 	old this vehicle<br/>
 	! 'This vehicle is old.'
 </div>
 
-<p>However, it can still be productive to use an explicit <span class="glyphs">da</span> <span class="toaq">da</span>: it serves as an unambiguous sentence boundary, and it can also be helpful for signaling to the listener that the current sentence is complete, especially when it isn't obvious from the structure of the sentence that the speaker does not intend to add more arguments.</p>
+<p>However, it can still be productive to use an explicit <span class="glyphs">󱚶󱚺</span> <span class="toaq">da</span>: it serves as an unambiguous sentence boundary, and it can also be helpful for signaling to the listener that the current sentence is complete, especially when it isn't obvious from the structure of the sentence that the speaker does not intend to add more arguments.</p>
 
-<p>When leaving <span class="glyphs">da</span> <span class="toaq">da</span> implicit, then, in order to avoid sentence boundary ambiguities, the following sentence should begin with a matrix complementizer (usually <span class="glyphs">va</span> <span class="toaq">ꝡa</span>), or a conjunction like <span class="glyphs">keo</span> <span class="toaq">keo</span> "but".</p>
+<p>When leaving <span class="glyphs">󱚶󱚺</span> <span class="toaq">da</span> implicit, then, in order to avoid sentence boundary ambiguities, the following sentence should begin with a matrix complementizer (usually <span class="glyphs">󱛁󱚺</span> <span class="toaq">ꝡa</span>), or a conjunction like <span class="glyphs">keo</span> <span class="toaq">󱛄󱚴󱛍󱛃</span> "but".</p>
```

Comment:
The Deranı and the romanization should be the other way around:
```suggestion
<p>When leaving <span class="glyphs">󱚶󱚺</span> <span class="toaq">da</span> implicit, then, in order to avoid sentence boundary ambiguities, the following sentence should begin with a matrix complementizer (usually <span class="glyphs">󱛁󱚺</span> <span class="toaq">ꝡa</span>), or a conjunction like <span class="glyphs">󱛄󱚴󱛍󱛃</span> <span class="toaq">keo</span> "but".</p>
```